tests/unit/test_redshift_spectrum.py

```diff
@@ -0,0 +1,68 @@
+import os
+
+import pandas as pd
+import pytest
+
+from viadot.sources import S3, RedshiftSpectrum
+
+TEST_DF = pd.DataFrame(
+    [
+        [0, "A"],
+        [1, "B"],
+        [2, "C"],
+    ],
+    columns=["col1", "col2"],
+)
+
+S3_BUCKET = os.environ.get("S3_BUCKET")
+TEST_SCHEMA = "raw_test"
+TEST_TABLE = "test_redshift_spectrum"
+
+
+@pytest.fixture(scope="session")
+def redshift(aws_config_key):
+    redshift = RedshiftSpectrum(config_key=aws_config_key)
+
+    yield redshift
+
+
+@pytest.fixture(scope="session")
+def s3(aws_config_key):
+    s3 = S3(config_key=aws_config_key)
+
+    yield s3
+
+
+def test_from_df(redshift):
+    redshift.from_df(
+        df=TEST_DF,
+        to_path=f"s3://datawerfen-159170848751291/nesso/{TEST_SCHEMA}/{TEST_TABLE}",
+        database=TEST_SCHEMA,
+        table=TEST_TABLE,
+    )
+
+    result = redshift.exists(
+        database=TEST_SCHEMA,
+        table=TEST_TABLE,
+    )
```

Comment:
Can you pls add a test for `exists()` before this one? Otherwise this test is not very trustworthy 😅

Reply:
[⚡️ Improve test assertion](https://github.com/dyvenia/viadot/pull/636/commits/2916fdaf3b1ae810b28ed9375f9a3c6e5c0953bd)

Reply:
[✅ Add test for RedshiftSpectrum.exists() method](https://github.com/dyvenia/viadot/pull/636/commits/bd0e4e356110595645717db6c65ddf54f467ccf4)

Reply:
Ah sry, I think you should also run exists() before running from_df(), so, asserst exist is False -> create table -> assert exists is True, otherwise this table could have been there all along, even before you ran `from_df()`. Also this exists test, the way it is right now... does that make sense? Does it *really* test something? :p